template/app/build.gradle.kts

```diff
@@ -133,6 +133,8 @@ dependencies {
     implementation("androidx.compose.foundation:foundation:${Versions.COMPOSE_VERSION}")
     implementation("androidx.compose.material:material:${Versions.COMPOSE_VERSION}")
 
+    runtimeOnly("androidx.fragment:fragment-ktx:${Versions.ANDROIDX_FRAGMENT_KTX_VERSION}")
```

Comment:
Any reason we use `runtimeOnly` instead of `implementation`? 💭

Reply:
@Tuubz We can use `runtimeOnly` to reduce the compile time, as this ktx can be use at Runtime
But it does not necessary in our template, I will use the implementation for consistency.